docs/reference/method/MongoDBDatabase-aggregate.txt

```diff
@@ -24,15 +24,105 @@ Definition
 
    .. code-block:: php
 
-      function aggregate(array $pipeline, array $options = []): Traversable
+      function aggregate(
+          array $pipeline,
+          array $options = []
+      ): Traversable
 
-   This method has the following parameters:
-
-   .. include:: /includes/apiargs/MongoDBDatabase-method-aggregate-param.rst
-
-   The ``$options`` parameter supports the following options:
+Parameters
+----------
 
-   .. include:: /includes/apiargs/MongoDBDatabase-method-aggregate-option.rst
+``$pipeline`` : array
+  Specifies an :manual:`aggregation pipeline </core/aggregation-pipeline>`
+  operation.
+
+``$options`` : array
+  An array specifying the desired options.
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 20 80
+
+     * - Name
+       - Type
+       - Description
+
+     * - allowDiskUse
+       - boolean
+       - Enables writing to temporary files. When set to ``true``, aggregation
+         stages can write data to the ``_tmp`` sub-directory in the ``dbPath``
+         directory.
+
+     * - batchSize
+       - integer
+       - Specifies the batch size for the cursor, which will apply to both the
+         initial ``aggregate`` command and any subsequent ``getMore`` commands.
+         This determines the maximum number of documents to return in each
+         response from the server.
+
+         A batchSize of ``0`` is special in that and will only apply to the
+         initial ``aggregate`` command; subsequent ``getMore`` commands will use
+         the server's default batch size. This may be useful for quickly
+         returning a cursor or failure from ``aggregate`` without doing
+         significant server-side work.
+
+     * - bypassDocumentValidation
+       - boolean
+       - If ``true``, allows the write operation to circumvent document level
+         validation. Defaults to ``false``.
+
+         This only applies when using the :ref:`$out <agg-out>` and
+         :ref:`$out <agg-merge>` stages.
+
+     * - collation
```

Comment:
The `codec` option is missing

Reply:
See https://github.com/mongodb/mongo-php-library/pull/1203#discussion_r1424415031